app/src/picovr/cpp/DeviceDelegatePicoVR.cpp

```diff
@@ -71,6 +71,15 @@ struct DeviceDelegatePicoVR::State {
     bool IsRightHand() const {
       return hand == ElbowModel::HandEnum::Right;
     }
+
+    void Reset() {
+        created = false;
+        enabled = false;
+        touched = false;
+        is6DoF = false;
+        transform = vrb::Matrix::Identity();
+        hand = ElbowModel::HandEnum::Right;
```

Comment:
The hand is currently set up in Initialize(). After Pause() & Resume() flow is called it looks like the ElbowModel will always be HandEnum::Right and never left.